examples/rex_planar_quadrotor_mpc.jl

```diff
@@ -18,13 +18,49 @@ import TrajectoryOptimization as TO
 using Altro
 import BilinearControl.Problems
 using Test
+using Infiltrator
 
 include("learned_models/edmd_utils.jl")
```

Comment:
shouldn't be using this file anymore. It should have been moved into `/src/edmd/`